packages/app/src/elements/import-dialog.ts

```diff
@@ -149,12 +150,22 @@ export class ImportDialog extends Dialog<File, void> {
                                     ?hidden=${itemColumn.type === "name" || itemColumn.type === "tags"}
                                 >
                                     <div class="small margined spacing horizontal layout">
-                                        <div class="stretch">${itemColumn.name}</div>
-                                        <div class="subtle">${$l("Column {0}", itemColumnIndex.toString())}</div>
+                                        <div class="stretch">
+                                            ${csvHasColumnsOnFirstRow
+                                                ? itemColumn.name
+                                                : $l("Column {0}", itemColumnIndex.toString())}
+                                        </div>
+                                        <div class="subtle" ?hidden=${!csvHasColumnsOnFirstRow}>
+                                            ${$l("Column {0}", itemColumnIndex.toString())}
+                                        </div>
                                     </div>
 
-                                    <div class="tiny horizontally-margined subtle mono">
-                                        ${itemColumn.exampleValues}
+                                    <div class="tiny horizontally-margined subtle mono ellipsis">
+                                        ${itemColumn.values
+                                            .filter((v) => v !== "")
+                                            .slice(0, 20)
+                                            .map((value) => (value.includes(",") ? `"${value}"` : value))
+                                            .join(", ")}
```

Comment:
Why limit the number of example values? Let's use all the space we have :)

Reply:
I just figured it could be confusing, but you're right.